tor/core/mentions.py

```diff
@@ -24,6 +24,9 @@ def process_mention(mention, r, tor, redis_server, config):
     :param redis_server: Active redis instance.
     :return: None.
     """
+
+    submission = r.submission(id=clean_id(mention.parent_id))
```

Comment:
So fun fact: if you call r.submission (instead of r.comment) on a comment, it will error when you try to interact with it, which is why we keep it after that if / else -- that way we know it's good to go.